Surface_mesh_parameterization/include/CGAL/Surface_mesh_parameterization/LSCM_parameterizer_3.h

```diff
@@ -77,10 +77,7 @@ namespace Surface_mesh_parameterization {
 /// \code
 ///   CGAL::Eigen_solver_traits<Eigen::SimplicialLDLT< Eigen::SparseMatrix<double> > >
 /// \endcode
-///         Otherwise, it uses CGAL's wrapping function to the OpenNL library:
-/// \code
-///   OpenNL::SymmetricLinearSolverTraits<typename TriangleMesh::NT>
-/// \endcode
+///
```

Comment:
It is good to remove the OpenNL.
The lines above the `\code` part of Eigen suggest that when Eigen is no enabled something else is used but what?